public/utils/wz-logo-menu.js

```diff
@@ -13,7 +13,7 @@
 // Remove Kibana Wazuh name and breadcrumb
 export const changeWazuhNavLogo = () => {
   const interval = setInterval(() => {
-    const nav = $('.euiHeader > .euiBreadcrumbs  > .euiBreadcrumb');
+    const nav = $('[data-test-subj="breadcrumbs"]  > .euiBreadcrumb');
```

Comment:
This is jquery? Wouldn't it be better to use vanilla js? and cut with the mix of technologies?